public.yml

```diff
@@ -29,7 +29,9 @@ allow-licenses:
   - 'MIT-0'
   - 'MIT AND ISC'
   - 'MIT AND Python-2.0'
+  - 'MIT AND NOASSERTION AND Python-2.0'
   - 'MIT-advertising'
+  - 'MPL-2.0'
```

Comment:
Hello @Mblanchetdeverly,

Is 'MPL-2.0' approved for public/distributed projects?

Reply:
No its not as it is a copyleft license. It could be approved for a specific use case, if the binaries are distributed but not the code, since the obligations only apply when using code, not when linking binaries. However, we cannot approve it at large.

Reply:
Hi @Mblanchetdeverly, in this case:

- This is introduced by the `pathspec` module ([link](https://github.com/cpburnz/python-pathspec/tree/master?tab=readme-ov-file))
- It is only used for testing (for formatting code, to be more precise)
- The library is not actually distributed with the product
- Users don't install the library when they install the app

Reply:
@jonapich then the MPL is fine for your specific use case. As mentionned before, I cannot approve it at large so we will not add it to the list of approved licenses but you can proceed in your case and ignore the alert with a comment "approved by Legal based on my specific use case".